lib/api/dsl/storage/index.js

```diff
@@ -477,22 +474,22 @@ class Storage {
    */
   _reindexTestTable (index, collection) {
     /*
-     * Since the reindexation has been triggered, the test table might have been
+     * Since the reindexation has been triggered, the test table &/|| filters might have been
      * destroyed
      */
     if (!this.testTables[index] || !this.testTables[index][collection] || !this.testTables[index][collection].reindexing) {
       return;
     }
-
-    const tt = this.testTables[index][collection];
-
-    /*
-     * If there is a huge increase in new subscriptions, we should postpone the
-     * reindexation to a later time.
-     */
-    if (tt.removedConditions.array.length < tt.clength / 10) {
+    if (!this.filtersIndex[index]) {
```

Comment:
This puzzles me. Next week I want to take a serious look at this reindexation code, there are too many behavior inconsistencies to my liking.